handler/src/main/java/io/netty/handler/ssl/SslContextBuilder.java

```diff
@@ -48,6 +51,17 @@ public static SslContextBuilder forServer(File keyCertChainFile, File keyFile) {
     /**
      * Creates a builder for new server-side {@link SslContext}.
      *
+     * @param key a PKCS#8 private key
+     * @param keyCertChain the X.509 certificate chain
+     * @see #keyManager(PrivateKey, X509Certificate[])
+     */
+    public static SslContextBuilder forServer(PrivateKey key, X509Certificate... keyCertChain) {
+        return new SslContextBuilder(true).keyManager(key, keyCertChain);
```

Comment:
@normanmaurer - Is it necessary to double check? Can we just leave the responsibility to check this in 1 place?